### ISSUE TITLE
[Bug]: can't search data immediately when i complete load(refresh=true) (diskann)（zilliz-cloud）

### ISSUE THREAD

**Issue by lentitude2tk** [milvus-io/milvus#22500]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.2.3
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2): java-sdk
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Import the same data for the loaded collection. The memory instance can search for data immediately after the import is completed, but the diskann instance can search for data about 1 minute after the import is completed. The search uses the search interface of the sdk。Diskann instance problem can be reproduced stably

### Expected Behavior

Import the same data for the loaded collection. Regardless of whether it is a memory instance or a diskann implementation, after the import is completed, the data can be searched immediately after executing load(refresh=true)

**Comment by lentitude2tk** [comment]
The test uses a json file close to 1GB, and the small file does not have this problem

**Comment by xiaofan-luan** [comment]
seems load refresh is not working as expected

**Comment by yanliang567** [comment]
@zhuwenxing  please keep an eye on this issue.

**Comment by xiaofan-luan** [comment]
@soothing-rain 
looks like refresh is not working?

**Comment by soothing-rain** [comment]
> @soothing-rain looks like refresh is not working?

We will look into this. Could be an issue with refresh. Could also be that index is not really complete despite what API reports.

**Comment by soothing-rain** [comment]
@lentitude2tk Can you verify if problems still exists after https://github.com/milvus-io/milvus/issues/22374

/assign @lentitude2tk 
/assign @yhmo 
/unassign

**Comment by lentitude2tk** [comment]
I will use the new kernel version for the next test @soothing-rain

**Comment by lentitude2tk** [comment]
I found out that the previous version of this link was merged into 2.2.0. I used the 2.2.3 version for the previous test. The merged content should have been included?

![image](https://user-images.githubusercontent.com/108672767/224644541-b8c6591a-dfbf-41eb-ac29-6a44c882bd23.png)

**Comment by xiaofan-luan** [comment]
> I found out that the previous version of this link was merged into 2.2.0. I used the 2.2.3 version for the previous test. The merged content should have been included?
> 
> ![image](https://user-images.githubusercontent.com/108672767/224644541-b8c6591a-dfbf-41eb-ac29-6a44c882bd23.png)

it's in 2.2.0 branch but it will actually released in 2.2.4 tag.

**Comment by lentitude2tk** [comment]
> > I found out that the previous version of this link was merged into 2.2.0. I used the 2.2.3 version for the previous test. The merged content should have been included?
> > ![image](https://user-images.githubusercontent.com/108672767/224644541-b8c6591a-dfbf-41eb-ac29-6a44c882bd23.png)
> 
> it's in 2.2.0 branch but it will actually released in 2.2.4 tag.

I did a test based on the 2.2.3 load(refresh=true) feature before. Has the modification of this [#22374](https://github.com/milvus-io/milvus/issues/22374) had been synchronized to the 2.2.3 or 2.2.4 related branches? Because I see that there is no 2.2.4 tag yet, it is not clear which branch I can verify the results based on。 Or I wait for the 2.2.4 tag to be pulled out before testing

**Comment by yanliang567** [comment]
currently branch 2.2.0 is our developing branch for 2.2.4, so you want to verify something new, please use a new tag in 2.2.0 branch.
